data_pipeline/algolia-update.js

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+const fs = require('fs');
+const dotenv = require('dotenv');
+const algoliasearch = require('algoliasearch');
+
+//load local environment variables
+const env = {
+  local: dotenv.parse(fs.readFileSync('.env.local'))
+};
+
+const charities = JSON.parse(
+  fs.readFileSync('data_pipeline/charity.json').toString('utf8')
+);
+
+const updatedAlgoliaIndex = [];
```

Comment:
```
const updatedAlgoliaIndex = Object.entries(charities).map(
  (result, [id, {category, charityname, servicetype]) => ({
    category,
    charityname,
    objectId: id,
    servicetype,
  )}
);
```
for the big kids

Reply:
mmm good call. [updated](https://github.com/CodeForBoulder/upswyng/pull/74/commits/f3552742c7ceb1074f2d9bbafb397f6bb8142c46)